doc/_templates/layout.html

```diff
@@ -2,7 +2,7 @@
 
 {% block menu %}
   {{ super() }}
-  <a href="cabal-projectindex.html">Reference</a>
-  <a href="genindex.html">Index</a>
+  <a href="cabal-projectindex.html">Quicklinks</a>
+  <a href="genindex.html">Word index</a>
```

Comment:
On “Quicklinks” vs. “Reference” I am ambivalent, but we need to understand what exactly the section is about. It is not a TOC, nor an index for sure.

Reply:
Ah, I see, we *already* have a `Cabal feference` section in the TOC, and this indeed seems a condensed version of `3. Cabal reference`. Let's see if people come up with a better name (“Cabal syntax quicklinks”?)

Reply:
I like "Cabal Syntax Quicklinks". Let's try this.